scripts/webpack/frontend.js

```diff
@@ -34,6 +35,9 @@ const commonConfigs = ({ platform }) => ({
         new webpack.DefinePlugin({
             'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
         }),
+        new FilterWarningsPlugin({ 
+            exclude: /export .* was not found in/ 
+        }),
```

Comment:
As per [this section](https://github.com/TypeStrong/ts-loader/blob/3cc78b8874848c4485250d084a7ffbe3b3fee4ff/README.md#transpileonly) of `ts-loader`'s README

> ... webpack 4 will give you "export not found" warnings any time you re-export a type

Eg.

```bash
$ "export 'OphanABEvent' was not found in '@guardian/types'
```

To prevent thiis problem we are disabling warnings using a regex `/export .* was not found in/` via the `webpack-filter-warnings-plugin`

### Why use this plugin?
Because after several hours of trying different config combinations I _could not_ get webpack 4 to stop showing these warnings. Only this lib seemed to work